homework/02LessonTwo.js

```diff
@@ -54,22 +75,22 @@ function lessThanOrEqualTo(parameter1, parameter2) {
 // What is a function? How do you define a function in Javascript?
 //
 // Your Answer Goes Here:
-//
+//    Takes parameters, does something with it, produces output.
 //
 // What is a return value?
 //
 // Your Answer Goes Here:
-//
+//    Return value is the output of the value of the function.
 //
 // How do you define a named function?
 //
 // Your Answer Goes Here:
-//
+//    By calling on the function (not sure about this)
```

Comment:
Thanks for the context! DEFINE a named function. check yo book https://intermediate.austincodingacademy.com/_book/lesson_two.html

Under "Named Functions" heading